src/PostgREST/CLI.hs

```diff
@@ -80,14 +80,20 @@ readCLIShowHelp =
   where
     prefs = O.prefs $ O.showHelpOnError <> O.showHelpOnEmpty
     opts = O.info parser $ O.fullDesc <> progDesc
-    parser = O.helper <*> exampleParser <*> cliParser
+    parser = O.helper <*> versionParser <*> exampleParser <*> cliParser
 
     progDesc =
       O.progDesc $
         "PostgREST "
         <> BS.unpack prettyVersion
         <> " / create a REST API to an existing Postgres database"
 
+    versionParser =
+      O.infoOption ("PostgREST " <> BS.unpack prettyVersion) $
+        O.long "version"
+        <> O.short 'v'
+        <> O.help "Show the version information"
+
```

Comment:
To keep consistency with the other flags:

```suggestion
    versionFlag =
      O.infoOption ("PostgREST " <> BS.unpack prettyVersion) $
        O.long "version"
        <> O.short 'v'
        <> O.help "Show the version information"

```

Also this doesn't seem to do parsing.